lighthouse-core/test/computed/metrics/cumulative-layout-shift-test.js

```diff
@@ -5,15 +5,18 @@
  */
 'use strict';
 
-const CumulativeLayoutShift = require('../../../computed/metrics/cumulative-layout-shift.js'); // eslint-disable-line max-len
-const jumpyClsTrace = require('../../fixtures/traces/jumpy-cls-m90.json');
-const oldMetricsTrace = require('../../fixtures/traces/frame-metrics-m89.json');
-const allFramesMetricsTrace = require('../../fixtures/traces/frame-metrics-m90.json');
-const preClsTrace = require('../../fixtures/traces/progressive-app-m60.json');
-const createTestTrace = require('../../create-test-trace.js');
-
 /* eslint-env jest */
 
+import CumulativeLayoutShift from '../../../computed/metrics/cumulative-layout-shift.js';
+import createTestTrace from '../../create-test-trace.js';
+import {readJson} from '../../../../root.js';
+
+const TRACE_FIXTURES = '../../fixtures/traces';
+const jumpyClsTrace = readJson(`${TRACE_FIXTURES}/jumpy-cls-m90.json`, import.meta);
```

Comment:
any reason these aren't imports?

Reply:
It was described in an earlier comment thread, but basically half way through I realized we really can do a plain import "because jest".